docs/reference/rbac.md

```diff
@@ -375,13 +375,13 @@ They have the following policies granted to them:
 
 Policy                                                  | Admins | SuperUsers | Developers| Viewers |
 --------------------------------------------------------|--------|------------|-----------|---------|
-[`FSFullAccess`](#fsfullaccess)                         |   ✅   |     ✅     |           |         |
-[`AuthFullAccess`](#authfullaccess)                     |   ✅   |            |           |         | 
-[`RepoManagementFullAccess`](#repomanagementfullaccess) |   ✅   |            |           |         | 
-[`AuthManageOwnCredentials`](#authmanageowncredentials) |        |     ✅     |    ✅     |   ✅    |
-[`RepoManagementReadAll`](#repomanagementreadall)       |        |     ✅     |    ✅     |         |
-[`FSReadWriteAll`](#fsreadwriteall)                     |        |            |    ✅     |         |
-[`FSReadAll`](#fsreadall)                               |        |            |           |   ✅    |
+[_FSFullAccess_](#fsfullaccess)                         |   ✅   |     ✅     |           |         |
+[_AuthFullAccess_](#authfullaccess)                     |   ✅   |            |           |         | 
+[_RepoManagementFullAccess_](#repomanagementfullaccess) |   ✅   |            |           |         | 
+[_AuthManageOwnCredentials_](#authmanageowncredentials) |        |     ✅     |    ✅     |   ✅    |
+[_RepoManagementReadAll_](#repomanagementreadall)       |        |     ✅     |    ✅     |         |
+[_FSReadWriteAll_](#fsreadwriteall)                     |        |            |    ✅     |         |
+[_FSReadAll_](#fsreadall)                               |        |            |           |   ✅    |
```

Comment:
Again, I don't understand the need for this. The policy names make sense as code.

Reply:
I just don't like code notation in links. WDYT?

Reply:
to my previous comment - we need this in a style guide, otherwise it's just an opinion ;) 

![thats-your-opinion](https://user-images.githubusercontent.com/3671582/233326501-6b4c62ec-6168-45ec-83dc-2937168f7fca.gif)

and more importantly - it'll just get reverted again in a few months when someone else takes a pass through a the docs :)

Reply:
I agree that a guideline would be great - I suggest referring to [Google's guidelines](https://developers.google.com/style), since composing our own manual would be the definition of an overkill.

Also, I think these kind of changes should be made in consensus, since there doesn't seem to be one here.